### PR TITLE
Performed Crud on AccreditationModel

### LINKED ISSUE #49: Add CURD operation to Accreditation model
create the following function for the Accreditation model:
- create
- update
- read
- remove

use `user.js` for reference

### PR DESCRIPTION
**Pull request by VinitChawda06**
fixes #49

### CODE REVIEW

File: models/accreditation.js [hunk 1]
```diff
@@ -8,6 +10,34 @@ const accreditationSchema = {
     dateofExpiry: {type: Date, required: true}
 };
 
-const AccreditationModel = new connector.model('Accreditation', accreditationSchema);
+const Accreditation = new connector.model('Accreditation', accreditationSchema);
```
Comment (TejasNair9977) [inline models/accreditation.js:13]: please rename this back to AccreditationModel

File: models/accreditation.js [hunk 2]
```diff
@@ -8,6 +10,34 @@ const accreditationSchema = {
...
 
-export default AccreditationModel;
\ No newline at end of file
+async function remove(filter) {
+    const res = await Accreditation.findOneAndDelete(filter);
```
Comment (TejasNair9977) [inline models/accreditation.js:16]: make this change in every line under it too

**Review by TejasNair9977 — CHANGES_REQUESTED** [review]
Just a few changes.

File: models/accreditation.js [hunk 3]
```diff
@@ -1,5 +1,7 @@
 import connector from "#models/databaseUtil";
+import { logger } from "#util";
```
Comment (Hitansh159) [inline models/accreditation.js:2]: remove logger

File: models/accreditation.js [hunk 4]
```diff
@@ -1,5 +1,7 @@
...
 
+connector.set("debug", true);
```
Comment (Hitansh159) [inline models/accreditation.js:4]: remove debug as well

**Review by Hitansh159 — CHANGES_REQUESTED** [review]

**Review by Hitansh159 — APPROVED** [review]

**Pull request merged**